### PR TITLE
fix: cleanup for 0.28.0

### PR DESCRIPTION
**Pull request by jonaro00**

### CODE REVIEW

File: resources/static-folder/src/lib.rs [hunk 1]
```diff
@@ -5,6 +5,10 @@ use serde::Serialize;
 use shuttle_service::{Error, Factory, ResourceBuilder, Type};
 
 #[derive(Serialize)]
+#[deprecated(
+    since = "0.27.0",
+    note = "Folder names can now be hard coded. https://docs.shuttle.rs/configuration/files"
```
Comment (iulianbarbu) [inline resources/static-folder/src/lib.rs:10]: nit: 
```suggestion
    note = "Folder names can now be hard coded: https://docs.shuttle.rs/configuration/files".
```

**Review by iulianbarbu — APPROVED** [review]

**Review by orhun — APPROVED** [review]

**Comment by jonaro00** [comment]
Waiting with merge until release is closer

File: resources/static-folder/Cargo.toml [hunk 2]
```diff
@@ -9,4 +10,4 @@ keywords = ["shuttle-service", "static-folder"]
 [dependencies]
 async-trait = "0.1.56"
 serde = { version = "1.0.148", features = ["derive"] }
-shuttle-service = { path = "../../service", version = "0.27.0" }
+shuttle-service = { path = "../../service", version = ">=0.27.0, <0.31.0" }
```
Comment (oddgrd) [inline resources/static-folder/Cargo.toml:13]: What is this change for?

**Review by oddgrd — APPROVED** [review]
LGTM! We're certain that nothing depends on openssl in cargo-shuttle now, right? We could run the QA jobs for mac and windows on this PR to ensure it works, but it's probably overkill.

File: resources/static-folder/Cargo.toml (hunk 2 above)
Comment (jonaro00) [reply resources/static-folder/Cargo.toml:13]: This way, the final release 0.28 of static folder will stay compatible (but with deprecation compile warning) until 0.32 OR resource breaking change, whichever happen first.

**Pull request merged**